sdk/react/widgets/VideoViewPlayPause.js

```diff
@@ -133,19 +146,52 @@ class VideoViewPlayPause extends React.Component {
     this.state.play.animationOpacity.setValue(0);
   };
 
+  _renderPlayPauseButton = () => {
+    if (this.state[PLAY].animationOpacity._value === 0) {
+      return this._renderButton(PAUSE);
+    }
+    return this._renderButton(PLAY);
+  };
+
   _renderButton = (name) => {
     const fontStyle = {fontSize: this.props.fontSize, fontFamily: this.props.icons[name].fontFamily};
     const opacity = {opacity: this.state[name].animationOpacity};
     const animate = {transform: [{scale: this.state[name].animationScale}]};
-    const buttonColor = {color: this.props.buttonColor == null? "white": this.props.buttonColor};
+    const buttonColor = {color: this.props.buttonColor == null ? "white" : this.props.buttonColor};
```

Comment:
should it be `this.props.buttonColor === null `?

Reply:
@rcvrgs For some reason === does not work here. It fails and changes color to black.

Reply:
@rcvrgs in current case for props we have `undefined` and `===` returns `false` on this check. I've changed it to `!!this.props.buttonColor`, it should work correctly.